lib/pages/company_site/eti/popups/exists_popup.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CompanySite
+  module ETI
+    class Table
+      class ExistsPopup < self
+        EXISTS = {
+          available: 'В наличии',
+          not_available: 'Нет в наличии',
+          order: 'Под заказ',
+          awaiting: 'Ожидается поступление',
+          none: 'Не указано'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.